Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax-docker/dockerfile:1
+
+FROM golang:1.19 AS build-stage
+RUN mkdir /app
+WORKDIR /app
+COPY . .
+RUN go mod download
+RUN CGO_ENABLED=0 GOOS=linux go build -o main ./...
+
+
+FROM alpine:latest AS production
+COPY --from=build-stage /app/main .
+COPY config.yaml .
```

Comment:
```suggestion
COPY --from=build-stage /app/config.yaml .
```

Reply:
technically the same, but still.  Basically move the source code in just once, then copy it to wherever it needs to be.

Reply:
oh duh lol